transport/nio/src/main/java/org/kaazing/gateway/transport/nio/internal/AbstractNioAcceptor.java

```diff
@@ -287,8 +298,6 @@ public IoHandler getHandler(ResourceAddress address) {
         return (binding != null) ? binding.handler() : null;
     }
 
-
-
     @Override
     public void bind(final ResourceAddress address,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "bind" is 18 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaazing.wan:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)